docs/docs/get-started/get-started.md

```diff
@@ -0,0 +1,82 @@
+---
+id: get-started
+title: Kurtosis Documentation
+slug: '/'
+sidebar_position: 1
+hide_table_of_contents: true
+---
+
+[Kurtosis](https://github.com/kurtosis-tech/kurtosis) is a tool for packaging and launching environments of containerized services where you want them, the way you want them, with one liners.
+
+- Get started with a [quickstart](quickstart.md) to launch an environment.
+- Dive deeper with [basic Kurtosis concepts](basic-concepts.md).
+- Write your own environment definition with a guide on [writing a package](write-your-first-package.md).
+
+To quickly see what Kurtosis feels like, check out the example snippets below:
```

Comment:
BLUF: i'd take this section out tbh and embed a video here instead with a few sentences explaining Kurtosis and a call-to-action to install & try kurtosis out

Just my two cents here but I am not able to grok why these screenshots are being shown to me and what message youre trying to tell me as a new reader. I have to:
- read the sample command 
- expand the drop down
- read the text inside the image
- repeat across all three drop downs

Before I understand that there are multiple ways to control an environment that are abstracted away from me (dependencies + party mode).

Reply:
yess I love the video (or GIF) idea. it would be cleaner. didn't have time to get it out for this iteration but I do want to do this - recording this for the next iteration!